docs/products/postgresql/howto/optimize-pg-slow-queries.md

```diff
@@ -1,13 +1,43 @@
 ---
-title: Optimize PostgreSQL® slow queries
+title: Optimize Aiven for PostgreSQL® slow queries
 ---
 
-PostgreSQL® gives you the ability to
-[indentify slow queries](/docs/products/postgresql/howto/identify-pg-slow-queries) using the `pg_stat_statements` view. The following article
-contains several common optimization techniques to improve slow running
-queries.
+import Insights from "@site/static/images/icons/performance.svg";
+import ActionsIcon from "@site/static/images/icons/more.svg";
 
-## Common slow queries optimizations
+Optimize slow queries to ensure maximum service performance. Use Aiven's AI capabilities or  `pg_stat_statements` to identify optimization opportunities.
+
+## Artificial intelligence recommendations
+
+Use Aiven's artificial intelligence (AI) capabilities to suggest optimizations to
+your databases and queries.
+
+Aiven considers various aspects to suggest optimization, for example query
+structure, table size, existing indexes and their cardinality, column types and
+sizes, the connections between the tables and columns in the query.
+
+### Display optimization recommendations
+
+To optimize a query:
+
+1. In the [Aiven Console](https://console.aiven.io/login), open your Aiven for PostgreSQL service.
+1. Click <Insights className="icon"/> **AI insights**.
+1. For the query of your choice, click **Optimize**.
+   A window displays the query.
```

Comment:
```suggestion
   The **Query optimization report** window opens and shows an optimization suggestion in the **New optimized query** section.
```

Reply:
Let's bring this up to a team sync. This very "DITA" approach is a time sink for little value in our case, we need to find ways to go faster, and skipping this kind of result step is an easy pick.

Reply:
Sure, let's do that :-) Just or the record, this is not a result step coz it says `For the query of your choice, click **Optimize**.`. And only afterwards, there is a comment you added `A window displays the query.` (still in the same step). What I proposed, I just suggested making your comment more detailed.

Reply:
I modified my suggestion for the sake of conciseness and simplicity but still a bit more meaningful I hope.